packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2822,26 +2822,56 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         // The call to fetch the snapshot is very expensive and not always needed.
```

Comment:
`ackHandle` argument is irrelevant for this function, so it should be removed.

Reply:
it is still interesting from telemetry perspective.